### PR TITLE
Add restrictions for ArchiveFilesV2, DeleteFilesV1, ExtractFilesV1

### PR DESCRIPTION
**Pull request by max-zaytsev**
**Task name**: ArchiveFilesV2, DeleteFilesV1, ExtractFilesV1

**Description**: added restrictions.

**Documentation changes required:** N

**Added unit tests:** N

**Attached related issue:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected - verified sanity scenarios for tasks, checked that setting of prohibited variables is not allowed for one of the tasks

### CODE REVIEW

File: Tasks/ArchiveFilesV2/task.json [hunk 1]
```diff
@@ -138,6 +138,14 @@
             "argumentFormat": ""
         }
     },
+    "restrictions": {
```
Comment (anatolybolshakov) [inline Tasks/ArchiveFilesV2/task.json:141]: Could you please bump task version also?

File: Tasks/ExtractFilesV1/task.json [hunk 2]
```diff
@@ -74,6 +74,14 @@
             "argumentFormat": ""
         }
     },
+    "restrictions": {
```
Comment (anatolybolshakov) [inline Tasks/ExtractFilesV1/task.json:77]: Same for ExtractFiles task

File: Tasks/ArchiveFilesV2/task.json (hunk 1 above)
Comment (max-zaytsev) [reply Tasks/ArchiveFilesV2/task.json:142]: Done

File: Tasks/ExtractFilesV1/task.json (hunk 2 above)
Comment (max-zaytsev) [reply Tasks/ExtractFilesV1/task.json:78]: Done

**Review by anatolybolshakov — APPROVED** [review]

**Review by egor-bryzgalov — APPROVED** [review]

**Pull request merged**